sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -98,6 +100,133 @@ public EditContactSteps(WebDriverHelpers webDriverHelpers) {
+          softly
+              .assertThat(aContact.getDescriptionOfHowContactTookPlace())
+              .isEqualToIgnoringCase(editedContact.getDescriptionOfHowContactTookPlace());
+          softly
+              .assertThat(aContact.getProhibitionToWork())
+              .isEqualToIgnoringCase(editedContact.getProhibitionToWork());
+          softly
+              .assertThat(aContact.getHomeBasedQuarantinePossible())
+              .isEqualToIgnoringCase(editedContact.getHomeBasedQuarantinePossible());
+          softly
+              .assertThat(aContact.getQuarantine())
+              .isEqualToIgnoringCase(editedContact.getQuarantine());
+          softly.assertThat(aContact.getHighPriority()).isEqualToIgnoringCase("on");
+          softly
+              .assertThat(aContact.getDiabetes())
+              .isEqualToIgnoringCase(editedContact.getDiabetes());
+          softly
+              .assertThat(aContact.getImmunodeficiencyIncludingHiv())
+              .isEqualToIgnoringCase(editedContact.getImmunodeficiencyIncludingHiv());
+          softly
+              .assertThat(aContact.getLiverDisease())
+              .isEqualToIgnoringCase(editedContact.getLiverDisease());
+          softly
+              .assertThat(aContact.getMalignancy())
+              .isEqualToIgnoringCase(editedContact.getMalignancy());
+          softly
+              .assertThat(aContact.getChronicPulmonaryDisease())
+              .isEqualToIgnoringCase(editedContact.getChronicPulmonaryDisease());
+          softly
+              .assertThat(aContact.getRenalDisease())
+              .isEqualToIgnoringCase(editedContact.getRenalDisease());
+          softly
+              .assertThat(aContact.getChronicNeurologicalNeuromuscularDisease())
+              .isEqualToIgnoringCase(editedContact.getChronicNeurologicalNeuromuscularDisease());
+          softly
+              .assertThat(aContact.getCardiovascularDiseaseIncludingHypertension())
+              .isEqualToIgnoringCase(editedContact.getCardiovascularDiseaseIncludingHypertension());
+          softly
+              .assertThat(aContact.getAdditionalRelevantPreexistingConditions())
+              .isEqualToIgnoringCase(editedContact.getAdditionalRelevantPreexistingConditions());
+          softly
+              .assertThat(aContact.getVaccinationStatusForThisDisease())
+              .isEqualToIgnoringCase(editedContact.getVaccinationStatusForThisDisease());
+          softly
+              .assertThat(aContact.getImmunosuppressiveTherapy())
+              .isEqualToIgnoringCase(editedContact.getImmunosuppressiveTherapy());
+          softly
+              .assertThat(aContact.getActiveInCare())
+              .isEqualToIgnoringCase(editedContact.getActiveInCare());
+          softly.assertThat(aContact.getOverwriteFollowUp()).isEqualToIgnoringCase("on");
+          // softly
+          //    .assertThat(aContact.getDateOfFollowUpUntil())
+          //    .isEqualTo(editedContact.getFollowUpStatusComment());
```

Comment:
Please remove commented code

Reply:
commented code is required to run when  the issue tagged will be fixed. see the feature file.